src/lib/pipeline_configuration.py

```diff
@@ -0,0 +1,620 @@
+import json
```

Comment:
(sorry this wasn't as copy/paste I'd hoped - I had moved this into configuration/ but didn't want to merge until I could test it on a new project. Next time...)

Reply:
Thanks will move well re arranging the scripts